### ISSUE TITLE
[nl.klikaanklikuit] KlikAanKlikUit ACDB-7000C No Longer Working

### ISSUE THREAD

**Issue by Veldkornet** [athombv/homey#2776]
**Metadata**

* **Homey Version**: 1.5.11
* **Homey Log Code**: 0D9F43825D
* **Smartphone Platform**:
* **Smartphone App Version**:

**If related to a Homey App:**
* **Homey App Version**: 4.0.11
* **Homey App Log Code**: ??
* **Affected Devices (preferably product codes)**: ACDB-7000C

**What did you try to achieve?**

Ring the doorbell

**What did you expect as result?**

Sound came out of the device

**What was the result?**

Nothing happened

**Any other remarks**
I've had a KAKU ACDB-7000C working with my Homey since I've had a Homey (about a year now). As of the last set of updates, it no longer works. I discovered this because the postman was bashing on my window instead of ringing the bell.
So I test it myself, nothing.
I try to add it again, nothing.
I completely reset the ACDB-7000C and try to re-pair, nothing.

So basically for the re-pair, I just follow the instructions. Set it up by letting it learn the code, then you push the learn button on the device, while the light is flashing you push the generate button in Homey, and then is goes to the next screen where you push the test button and...... nothing happens. No sound comes out of the device.
I've even put the device 15cm away from Homey, nothing. I'm pretty sure that it was working in the last week or two, so I guess it must be one of the last two updates that broke it.

If it wasn't bad enough that you guys all but [broke my doorbell button](https://github.com/athombv/com.fibaro/issues/198), you've now screwed up the bell itself as well...

**Comment by Veldkornet** [comment]
FYI, i managed to fix the issue myself by restarting the microcontroller multiple times and then repeating the steals above until it eventually worked. 

of course, that in itself doesn’t sound too healthy to me.

**Comment by Dijker** [comment]
Belongs to the category #2023 

In all cases with trouble on 433: Pull the Plug from Homey! 
Should work 1000 times better than restart Microcontroller.

will be fixed in one of the next updates read here: https://blog.athom.com/behind-magic-july-2018/
